reana_job_controller/htcondorcern_job_manager.py

```diff
@@ -53,8 +51,9 @@ def __init__(
         unpacked_img=False,
         htcondor_max_runtime="",
         htcondor_accounting_group=None,
+        **kwargs,
```

Comment:
Added `kwargs` here so, in case, `kubernetes_job_timeout` or another unsupported parameter will be passed it will be ignored.